java/plasma/test.sh

```diff
@@ -47,10 +47,10 @@ pushd ../../cpp
             -DARROW_WITH_ZSTD=off \
             -DARROW_PLASMA_JAVA_CLIENT=on \
             ..
-        make VERBOSE=1 -j$PARALLEL
+        sudo make VERBOSE=1 -j$PARALLEL
     popd
 popd
 
 mvn clean install
-export PLASMA_STORE=$ROOT_DIR/../../cpp/release/release/plasma_store_server
-java -cp target/test-classes:target/classes -Djava.library.path=$ROOT_DIR/../../cpp/release/release/ org.apache.arrow.plasma.PlasmaClientTest
+export PLASMA_STORE=$CONDA_PREFIX/tmp/arrow/cpp/release/release/plasma-store-server
```

Comment:
Its not clear to me that CONDA_PREFIX is the right thing to use here if this is intended as a development script.